pyradiosky/skymodel.py

```diff
@@ -399,6 +398,8 @@ def healpix_to_sky(hpmap, indices, freqs):
     -----
     Currently, this function only converts a HEALPix map with a frequency axis.
     """
+    import astropy_healpix
```

Comment:
Would it be better to have this fail with a message like, "astropy-healpix module required to use HEALPix methods"?

Reply:
yes, I'll add that.